libs/api/domains/document-provider/src/lib/document-provider.service.ts

```diff
@@ -241,14 +235,23 @@ export class DocumentProviderService {
   }
 
   async updateEndpointOnTest(
+    nationalId: string,
     endpoint: string,
     providerId: string,
     xroad: boolean,
+    authorization: Auth,
   ): Promise<AudienceAndScope> {
-    // return new AudienceAndScope(
-    //   'https://test-skjalaveita-island-is.azurewebsites.net',
-    //   'https://test-skjalaveita-island-is.azurewebsites.net/api/v1/customer/.default',
-    // )
+    const isLastModifier = await this.isLastModifierOfOrganisation(
+      nationalId,
+      authorization,
+    )
+
+    if (!isLastModifier) {
+      throw new ApolloError(
+        'Forbidden. User is not last modifier of organisation.',
+        '403',
+      )
+    }
```

Comment:
As we are re-using this around this file we should move this into a helper function.